app/assets/scripts/components/project/sec-panel/class-analytics-chart.js

```diff
@@ -139,34 +138,30 @@ function ClassAnalyticsChart(props) {
         />
       </ChartContainer>
       <Summary>
-        {bounds && (
-          <ClassItem bounds={bounds}>
-            <p> </p>
-            <Prose size='small'>CLASS NAME</Prose>
-            <Prose size='small' className='percent'>
-              AREA KM2
-            </Prose>
-            <Prose size='small' className='percent'>
-              %
-            </Prose>
-          </ClassItem>
-        )}
+        <ClassItem>
```

Comment:
```suggestion
{ totalArea &&        <ClassItem>
```
reinstate conditional rendering of this item that functions as a table header - unneeded for the F1 chart